### ISSUE TITLE
Arduino code crashing MoppyDesk?

### ISSUE THREAD

**Issue by ApexdaGamer** [samphonic/MoppyClassic#205]
I can't get the code to work on my Arduino Mega2560 R3. It seems I have everything right and ready, it's just that as soon as I connect MoppyDesk the application closes with Java Error code 1. I've been searching around on the error, and I've found that it's not because my storage, as I have plenty. I don't know why, but right now I'm having some big issues. I cannot use Moppy2 because Apache NetBeans doesn't detect it as a Project Folder, and if it's for the other IDE (Eclipse) I dunno how to use it.

**Comment by samphonic** [comment]
Yeah, MoppyClassic is getting pretty out of date, so there's not too much I can do to help you with that.

As for Moppy2, if you just download [the latest release](https://github.com/Sammy1Am/Moppy2/releases/tag/2.1.0), you don't actually need NetBeans anyway (unless you want to modify the code for some reason).  Just open the `.bat` file (if you're on Windows) to run it.  If it crashes, run the `.bat` from a command prompt, but 9 times out of 10 it's because JAVA_HOME needs to be properly set.
